zerver/actions/create_realm.py

```diff
@@ -188,7 +187,19 @@ def do_create_realm(
             event_time=realm.date_created,
         )
 
-        RealmUserDefault.objects.create(realm=realm)
+        realm_default_email_address_visibility = RealmUserDefault.EMAIL_ADDRESS_VISIBILITY_EVERYONE
+        if (
+            realm.org_type == Realm.ORG_TYPES["education_nonprofit"]["id"]
+            or realm.org_type == Realm.ORG_TYPES["education"]["id"]
+        ):
+            # Email address of users should be initially visible to admins only.
+            realm_default_email_address_visibility = (
+                RealmUserDefault.EMAIL_ADDRESS_VISIBILITY_ADMINS
+            )
```

Comment:
This is not a new feature. Previously we set the realm `email_address_visibility` setting for the education type orgs to be `Admins only`. Here as we will remove the realm-level setting, we set the realm-level defaut to be `Admins only` so that users when joining will have that value.

Reply:
Ahh, OK, that makes sense -- I was just looking in the wrong place for it being removed.